modin/backends/base/query_compiler.py

```diff
@@ -1740,218 +3577,1371 @@ def set_index_names(self, names, axis=0):
+        arg : callable(pandas.Resampler) -> pandas.Series
+        *args : iterable
+            Positional arguments to pass to function.
+        **kwargs : dict
+            Keyword arguments to pass to function.
+
+        Returns
+        -------
+        BaseQueryCompiler
+            New QueryCompiler containing the result of passed function.
+        """
+        return ResampleDefault.register(pandas.core.resample.Resampler.transform)(
+            self, resample_args, arg, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(result="variance", params="ddof : int", link="var")
+    def resample_var(self, resample_args, ddof, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.var)(
+            self, resample_args, ddof, *args, **kwargs
+        )
+
+    # End of Resample methods
+
+    # Str methods
+
+    @_doc_str_method(method="capitalize", params="")
+    def str_capitalize(self):
+        return StrDefault.register(pandas.Series.str.capitalize)(self)
+
+    @_doc_str_method(
+        method="center",
+        params="""width : int
+        fillchar : str, default: ' '""",
+    )
+    def str_center(self, width, fillchar=" "):
+        return StrDefault.register(pandas.Series.str.center)(self, width, fillchar)
+
+    @_doc_str_method(
+        method="contains",
+        params="""pat : str
+        case : bool, default: True
+        flags : int, default: 0
+        na : object, default: np.NaN
+        regex : bool, default: True""",
+    )
+    def str_contains(self, pat, case=True, flags=0, na=np.NaN, regex=True):
+        return StrDefault.register(pandas.Series.str.contains)(
+            self, pat, case, flags, na, regex
+        )
+
+    @_doc_str_method(
+        method="count",
+        params="""pat : str
+        flags : int, default: 0
+        **kwargs : dict""",
+    )
+    def str_count(self, pat, flags=0, **kwargs):
+        return StrDefault.register(pandas.Series.str.count)(self, pat, flags, **kwargs)
+
+    @_doc_str_method(
+        method="endswith",
+        params="""pat : str
+        na : object, default: np.NaN""",
+    )
+    def str_endswith(self, pat, na=np.NaN):
+        return StrDefault.register(pandas.Series.str.endswith)(self, pat, na)
+
+    @_doc_str_method(
+        method="find",
+        params="""sub : str
+        start : int, default: 0
+        end : int, optional""",
+    )
+    def str_find(self, sub, start=0, end=None):
+        return StrDefault.register(pandas.Series.str.find)(self, sub, start, end)
+
+    @_doc_str_method(
+        method="findall",
+        params="""pat : str
+        flags : int, default: 0
+        **kwargs : dict""",
+    )
+    def str_findall(self, pat, flags=0, **kwargs):
+        return StrDefault.register(pandas.Series.str.findall)(
+            self, pat, flags, **kwargs
+        )
+
+    @_doc_str_method(method="get", params="i : int")
```

Comment:
`i` could use a description IMHO, not only a type

Reply:
this one is also has a link to the pandas docs with the description of the parameter